snooty/types.py

```diff
@@ -401,7 +402,7 @@ def open(cls, root: Path) -> Tuple["ProjectConfig", List[Diagnostic]]:
 
             path = path.parent
 
-        return cls(root, "untitled"), diagnostics
+        return cls(root, name="unnamed", title="untitled"), diagnostics
```

Comment:
Because `title` has a default value, you don't actually need to pass that parameter